Jie/motivators

jiexiangfan opened the pull request:
- Implemented checkbox group in custom field for users to add motivators/demotivators
- Motive metrics shows the motivators and demotivators as two different radar charts
- The page is using MUI for consistency
- The radar chart is configured so that axes start from zero and the ticks increase by 1
- The colour of radar chart is set to 0.5 opacity to not block the grids

![Screen Shot 2022-09-29 at 15 37 12 pm](https://user-images.githubusercontent.com/51945335/192947942-a2d5bb3b-92a4-4caf-a4ed-3e58b2b7adc0.png)
![Screen Shot 2022-09-29 at 15 37 21 pm](https://user-images.githubusercontent.com/51945335/192947948-b44746d3-a1e0-4b8e-9df1-9e1719cc8bad.png)
![Screen Shot 2022-09-29 at 15 37 41 pm](https://user-images.githubusercontent.com/51945335/192947953-c1b246b2-4d49-42b0-8385-faaccfa72438.png)

jiexiangfan committed:
Merge branch 'master' into jie/motivators

jiexiangfan committed:
auto linting

jiexiangfan committed:
add check box group for mot/demot

jiexiangfan committed:
motivator -> demotivator

jiexiangfan committed:
tested and fixed ui

jiexiangfan committed:
comment output

jiexiangfan committed:
typo is d11

jiexiangfan committed:
add mot/demot titles to display on chart

jiexiangfan committed:
should work but data in Motivation.js empty

jiexiangfan committed:
added import back

jiexiangfan committed:
debugging comment

jiexiangfan committed:
field -> `${field}`

jiexiangfan committed:
console log

andrwcao committed:
Handled motivation data null cases

jiexiangfan committed:
Merge branch 'master' into jie/motivators

jiexiangfan committed:
remove console.log

jiexiangfan committed:
transparent background not to block the chart

jiexiangfan committed:
radarChart scaled beginAtZero with tick of 1

jiexiangfan committed:
Merge branch 'master' into jie/motivators

jiexiangfan committed:
styled motivation page with MUI

jiexiangfan committed:
check project id according https://github.com/Motive-Metrics/motive-metrics/pull/8#discussion_r981151560

andrwcao reviewed (approved):
Works well on my instance. Lots of formatting changes so assume all accurate. Would be good to have demotivators graph use a red colour? Either way, all goods

kdav108 commented:
Looking great! Support Andrew's idea to make demotivates graph in red, also is it "de-motivators" or "demotivators"? Most articles on Google seems to use the latter but error correcting underlines that word for me here.

Also not sure how hard it would be but can we make like 2 tabs for selecting motivators/demotivators, I'm just concerned whether people just read the top heading "Select motivator(s) of this ticker (if any)", and if they don't want to list any they may not scroll or look down to see the list for demotivators. Not a big deal, so if it's time consuming no point pursuing, rest is looking great!

jiexiangfan committed:
Merge branch 'master' into jie/motivators

jiexiangfan committed:
radar chart diff colour for motivators and demotivators

jiexiangfan committed:
de-motiv -> demotiv

jiexiangfan commented:
Sorry for my buggy formatter 😓 . I've changed the radar chart colour so that it shows red for demotivators and blue for motivators. And also changed "de-motivators" to "demotivators" (it was underlined on mine as well that's why I used de-motivators before lol 😂 ). 

I like the idea of using 2 tabs for demotivators and motivators selection, but it gets a bit tricky when using Atlassian UIKit. I will play around with it, and maybe add the changes to this PR or raise another PR for it.

Colours:
![image](https://user-images.githubusercontent.com/51945335/193448014-598c4937-5ede-493e-ab29-a04b53a9aab7.png)

jiexiangfan committed:
double quote -> single quote
